compiler-core/src/format.rs

```diff
@@ -26,7 +29,12 @@ use camino::Utf8Path;
 
 const INDENT: isize = 2;
 
-pub fn pretty(writer: &mut impl Utf8Writer, src: &EcoString, path: &Utf8Path) -> Result<()> {
+pub fn pretty(
+    writer: &mut impl Utf8Writer,
+    src: &EcoString,
+    path: &Utf8Path,
+    source_map_emitter: &mut SourceMapEmitter,
```

Comment:
Formatting never uses a source map emitter, there should be no change to this API.